### ISSUE TITLE
adaptative Learning Rate?

### ISSUE THREAD

**Issue by dariogonle** [ultralytics/yolov5#2819]
## ❔Question
Hi,

 I'd like to know how to train starting with a LR of 0.01, change to 0.001 at epoch 50 and finish at epoch 100 with a LR of 0.0001.

I guess I have to set up the hyp.yaml file. I'm sure that lr0 = 0.01, but I don't know what valur should I use for lrf.

Does the warmup affect in any way?

Thank you.

**Comment by glenn-jocher** [comment]
@dariogonle warmup gradually brings up LR from initial values (i.e. 0) to `hyp['lr0']`. You can modify the pytorch LR scheduler used in train.py here using a custom schedule of your own:
https://github.com/ultralytics/yolov5/blob/aff03be35a8f5c7fb7da8bfd2f26a93cde416fbc/train.py#L134-L142
